test/unit/logging.html

```diff
@@ -85,6 +85,22 @@
       assert.includeMembers(output, args, `${args} should be in ${output}`);
     });
   });
+  suite('_logger', () => {
+    test('_log with single parameter', () => {
+      const msg = 'log test';
+      const logHistory = [];
+      const orgConsole = console;      
+      window.console = {
```

Comment:
You can do the following with `sinon` (http://sinonjs.org/releases/v4.1.3/spies/):

```js
test('_log with single parameter', () => {
  const msg = 'log test';
  const spy = sinon.spy();
  const oldConsole = window.console;

  window.console = spy;
  Polymer.Base._log(msg);
  window.console = oldConsole;

  assert.isTrue(spy.calledOnce);
  assert.isTrue(spy.calledWith(msg));
});
```

Reply:
@TimvdLippe Any pros for this case?

Reply:
For consistency, as we use it throughout our testing suite: https://github.com/Polymer/polymer/search?utf8=%E2%9C%93&q=sinon.spy&type=